products.d/agama-products.spec

```diff
@@ -52,12 +66,16 @@ Definition of openSUSE products (Tumbleweed, Leap, MicroOS and Slowroll) for the
 %license LICENSE
 %dir %{_datadir}/agama
 %dir %{_datadir}/agama/products.d
+%if 0%{?is_opensuse} && 0%{?suse_version} > 1600
 %{_datadir}/agama/products.d/microos.yaml
 %{_datadir}/agama/products.d/tumbleweed.yaml
-%{_datadir}/agama/products.d/leap_160.yaml
-%{_datadir}/agama/products.d/leap_micro_62.yaml
 %{_datadir}/agama/products.d/slowroll.yaml
 %{_datadir}/agama/products.d/kalpa.yaml
+%endif
+%if 0%{?is_opensuse} && 0%{?suse_version} >= 1600
```

Comment:
this cause failure at https://build.opensuse.org/package/live_build_log/systemsmanagement:Agama:Devel/agama-products/openSUSE_Tumbleweed/x86_64

basically above it removes leap for > 1600, but here it is included. it should be `%if 0%{?is_opensuse} && 0%{?suse_version} > 1600`